step10/README.md

```diff
@@ -0,0 +1,90 @@
+# Step 08: Command line parameters
```

Comment:
Step 09 + somewhat funny title :)

(although step 8 has a boring title, I know...)

Reply:
Went with "Buffer The String Concatenation Slayer" for the attempt at a funny title 😀

Reply:
Tbh it took me a while to catch the Buffy reference, but it was a nice touch! :D 

Defintely keeping it! :)